x-pack/plugins/fleet/public/components/transform_install_as_current_user_callout.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { uniqBy } from 'lodash';
+
+import type { PackageInfo } from '../../common';
+
+export const getNumTransformAssets = (assets?: PackageInfo['assets']) => {
+  if (
+    !assets ||
+    !(Array.isArray(assets.elasticsearch?.transform) && assets.elasticsearch?.transform?.length > 0)
+  ) {
+    return 0;
+  }
+
+  return uniqBy(assets.elasticsearch?.transform, 'file').length;
+};
+export const TransformInstallWithCurrentUserPermissionCallout: React.FunctionComponent<{
+  count: number;
+}> = ({ count }) => {
+  return (
+    <EuiCallOut color="primary">
```

Comment:
I think callouts should always have a title with an icon , check out the EUI examples https://elastic.github.io/eui/#/display/callout

Reply:
Updated here [`0187c5f` (#154665)](https://github.com/elastic/kibana/pull/154665/commits/0187c5f05136799f17c5b139d16206a44dbf7c55)